### PR TITLE
Add the solipsist way

### PR DESCRIPTION
**Pull request by avalonv**

### CODE REVIEW

**Comment by hakluke** [comment]
I'm confused about this one @avalonv ! can you eli5?

**Comment by avalonv** [comment]
@hakluke I won't be upset if you don't merge this. it doesn't technically qualify because it doesn't actually exit vim, but gives you the impression that you did by taking you back to the place you were if you opened vim in a unix terminal.

it's called the solipsistit way because it's a bit like the matrix, you are always living in a dream, a simulation of reality, you can never truly escape vim

**Comment by avalonv** [comment]
Something I should've noted before submitting this is I only tested this in neovim. In vim `:terminal` enters insert mode automatically and splits the window in two, making it much less convincing.
